### PR TITLE
Update release notes for 19.4

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

File: source/partials/next_release/_release.md.erb [hunk 1]
```diff
@@ -1,23 +1,48 @@
-<h4>Some feature</h4>
+<h4>Introduction of new attributes</h4>
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+Introduced `username` and `password` (or `encryptedPassword`) attributes on Git and Mercurial materials. Users can now use these fields to setup credentials in a secure manner, instead of putting credentials in the URL. The YAML, JSON and Groovy config repo plugins have been updated to allow these additional fields on material definitions.
```
Comment (bdpiparva) [inline source/partials/next_release/_release.md.erb]: `The YAML, JSON and Groovy config repo plugins` =>  please add a link to the plugin?

File: source/partials/next_release/_release.md.erb [hunk 2]
```diff
@@ -1,23 +1,48 @@
...
+Users can also utilize the newly introduced `branch` attribute on Mercurial materials tp specify the branch instead of specifying in the URL.
```
Comment (bdpiparva) [inline source/partials/next_release/_release.md.erb]: `tp specify ` => `to specify`

File: source/partials/next_release/_release.md.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<h4>Some feature</h4>
+<h4>Introduction of new attributes</h4>
 
 Introduced `username` and `password` (or `encryptedPassword`) attributes on Git and Mercurial materials. Users can now use these fields to setup credentials in a secure manner, instead of putting credentials in the URL. The YAML, JSON and Groovy config repo plugins have been updated to allow these additional fields on material definitions.
 Users can also utilize the newly introduced `branch` attribute on Mercurial materials tp specify the branch instead of specifying in the URL.
```
Comment (adityasood) [inline source/partials/next_release/_release.md.erb]: Mercurial materials **to**

File: source/partials/next_release/_release.md.erb [hunk 4]
```diff
@@ -10,6 +10,12 @@ Users can also utilize the newly introduced `branch` attribute on Mercurial mate
 **WARNING:**
 * Changing the material config will be treated as a new material by the GoCD which might trigger other pipelines with the same material configs.
 
+<h4>Java support changes</h4>
```
Comment (adityasood) [inline source/partials/next_release/_release.md.erb]: Support for Java 12

File: source/partials/next_release/_release.md.erb [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<h4>Some feature</h4>
+<h4>Introduction of new attributes</h4>
```
Comment (adityasood) [inline source/partials/next_release/_release.md.erb]: `Improved configurations for git and hg material credentials`

File: source/partials/next_release/_release.md.erb [hunk 6]
```diff
@@ -36,6 +42,7 @@ Users can also utilize the newly introduced `branch` attribute on Mercurial mate
 
 * <%= deprecated_api_message api_name: 'Pipeline config', old_api_version: 6, new_api_version: 7, new_api_available_in_release: '19.4.0', api_section: 'pipeline-config', removal_date_and_year: 'Aug 2019' %>
 * <%= deprecated_api_message api_name: 'Config repo', old_api_version: 1, new_api_version: 2, new_api_available_in_release: '19.4.0', api_section: 'config-repo', removal_date_and_year: 'Aug 2019' %>
+* The support for Java 8 has been deprecated.
```
Comment (adityasood) [inline source/partials/next_release/_release.md.erb]: Support for Java 8 has been deprecated. The support for the same will be removed in a release scheduled for November 2018

**Comment by adityasood** [comment]
LGTM

**Pull request merged**